### PR TITLE
migrated from glide to dep

### PR DESCRIPTION
**Pull request by robbert229**
In this pull request I have migrated from glide to dep. #2997

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Gopkg.toml example
```
Comment (thomastaylor312) [inline Gopkg.toml]: Can you remove this example comment section?

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
One small change and then I'll approve. Thanks for taking this on!

**Comment by thomastaylor312** [comment]
@bacongobbler You may want to take a look at this since you were helping out with the first pass. @mattfarina I think it would be good for you to look at it too because you are an expert at this stuff

**Comment by robbert229** [comment]
I will remove the examples once i get home!

**Comment by robbert229** [comment]
@thomastaylor312 the examples have been removed.

**Comment by bacongobbler** [comment]
Thanks! I'll take a look at this next week. We'll need to take a *really* close look with this one. :)

File: glide.yaml [hunk 2]
```diff
@@ -1,73 +0,0 @@
...
-- package: golang.org/x/net
-  subpackages:
-  - context
-- package: github.com/spf13/cobra
-  version: f62e98d28ab7ad31d707ba837a966378465c7b57
-- package: github.com/spf13/pflag
-  version: 9ff6c6923cfffbcd502984b8e0c80539a94968b7
-- package: github.com/Masterminds/vcs
-  version: ~1.11.0
-  # Pin version of mergo that is compatible with both sprig and Kubernetes
```
Comment (adamreese) [inline glide.yaml]: This is important! This will need to be pinned in Gopkg.toml mergo

File: glide.yaml [hunk 3]
```diff
@@ -1,73 +0,0 @@
...
-  - ptypes/timestamp
-- package: google.golang.org/grpc
-  version: 1.7.2
-- package: github.com/gosuri/uitable
-- package: github.com/asaskevich/govalidator
-  version: ^4.0.0
-- package: golang.org/x/crypto
-  subpackages:
-  - openpgp
-# pin version of golang.org/x/sys that is compatible with golang.org/x/crypto
```
Comment (adamreese) [inline glide.yaml]: Need to validate if this is still an issue

**Review by adamreese — CHANGES_REQUESTED** [review]

**Comment by mrueg** [comment]
It would be great to see this in, since go-1.10 and glide cause this issue for me when trying to build helm from source: https://github.com/Masterminds/glide/issues/975

**Comment by rajatjindal** [comment]
@robbert229 can you please update the PR with requested changes?

**Comment by bacongobbler** [comment]
in case anyone wants to pick this up on the Helm 2 side, here's the Helm 3 PR: https://github.com/kubernetes/helm/pull/3945

**Comment by bacongobbler** [comment]
closing due to inactivity. We've already implemented this for Helm 3. Thanks!

**Pull request closed without merging**